ui-v2/tests/unit/services/atob-test.js

```diff
@@ -1,12 +1,12 @@
-import { moduleFor, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 
-moduleFor('service:atob', 'Unit | Service | atob', {
-  // Specify the other units that are required for this test.
-  // needs: ['service:foo']
-});
+module('Unit | Service | atob', function(hooks) {
+  setupTest(hooks);
 
-// Replace this with your real tests.
-test('it exists', function(assert) {
-  let service = this.subject();
-  assert.ok(service);
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let service = this.owner.lookup('service:atob');
+    assert.ok(service);
+  });
```

Comment:
same for service tests… I do find these useful on occasion, but there doesn’t seem to me to be much point in having the “it exists” ones.